### ISSUE TITLE
Добавить список выполненных задач

### ISSUE THREAD

**Issue by vavsar** [alpha-coders-team/todo-app#56]
Сейчас задачи исчезают из списка (получают статус "архив") без возможности их видеть и вернуть в список задач обратно.
Это неудобно. Нужно вывести список выполненных задач и сделать возможность изменить их статус на невыполненные

**Comment by Gaikanomer9** [comment]
Согласен, нужно только определиться как мы хотим выводить эти задачи. Есть вариант добавить справа сверху рядом с иконкой пользователя счетчик выполненных задач (0/10), например, и по клику открывать выполненные задачи сегодня в выпадающем списке с возможностью вернуть их в главный список дел

**Comment by vavsar** [comment]
Я видел это как список, отделенный чертой от первого, сделанный в формате перечеркнутых надписей

**Comment by vavsar** [comment]
Предлагаю в модель Таска добавить чекбокс - архив. И на главной два списка: первый не из архива, второй из архива. Это очень просто и быстро реализовать
